templates/repo/issue/card.tmpl

```diff
@@ -0,0 +1,66 @@
+{{with .Issue}}
+	{{if eq $.Page.Project.CardType 1}}{{/* Images and Text*/}}
+		<div class="card-attachment-images">
+			{{range (index $.Page.issuesAttachmentMap .ID)}}
+				<img src="{{.DownloadURL}}" alt="{{.Name}}" />
+			{{end}}
+		</div>
+	{{end}}
+	<div class="content gt-p-0">
+		<div class="gt-df gt-items-start">
+			<div class="issue-card-icon">
+				{{template "shared/issueicon" .}}
+			</div>
+			<a class="issue-card-title muted issue-title" href="{{.Link}}">{{.Title | RenderEmoji ctx | RenderCodeBlock}}</a>
+			{{if and $.isPinned $.Page.isRepoAdmin}}
+				<a role="button" class="issue-card-unpin muted gt-df gt-ac" data-tooltip-content={{ctx.Locale.Tr "repo.issues.unpin_issue"}} data-issue-id="{{.ID}}" data-unpin-url="{{$.Page.Link}}/unpin/{{.Index}}">
+					{{svg "octicon-x" 16}}
+				</a>
+			{{end}}
+		</div>
+		<div class="meta gt-my-2">
+			<span class="text light grey muted-links">
+				{{if not $.Page.Repository}}{{.Repo.FullName}}{{end}}#{{.Index}}
```

Comment:
this check is weird to read:
`if there is no repository, use another repository. Otherwise do nothing`?

Reply:
Do you have a better suggestion? Basically it assumes: if we have no Repository parameter on Page level, assume that this page can show issues from multiple repositories and display the repo name on the issue card